docs/docs/workflows/create.md

```diff
@@ -36,217 +36,19 @@ constellation create
-    initial_count = \"$(yq '.nodeGroups.worker_default.initialCount' constellation-conf.yaml)\"
-    }
-}" >> terraform.tfvars
-echo "project = \"$(yq '.provider.gcp.project' constellation-conf.yaml)\"" >> terraform.tfvars
-echo "region = \"$(yq '.provider.gcp.region' constellation-conf.yaml)\"" >> terraform.tfvars
-echo "zone = \"$(yq '.provider.gcp.zone' constellation-conf.yaml)\"" >> terraform.tfvars
-terraform fmt terraform.tfvars
-```
-
-</tabItem>
-</tabs>
-
-Initialize and apply Terraform to create the configured infrastructure:
-
-```bash
-terraform init
-terraform apply
-```
-
-The Constellation [apply step](#the-apply-step) requires the already created `constellation-config.yaml` and the `constellation-state.yaml`.
-Create the `constellation-state.yaml` using the output from the Terraform state and the `constellation-conf.yaml`:
-
-<tabs groupId="csp">
-<tabItem value="aws" label="AWS">
-
-```bash
-yq eval '.version ="v1"' --inplace constellation-state.yaml
-yq eval ".infrastructure.initSecret =\"$(terraform output initSecret | jq -r | tr -d '\n' | hexdump -ve '/1 "%02x"' && echo '')\"" --inplace constellation-state.yaml
-yq eval ".infrastructure.clusterEndpoint =\"$(terraform output out_of_cluster_endpoint | jq -r)\"" --inplace constellation-state.yaml
-yq eval ".infrastructure.inClusterEndpoint =\"$(terraform output in_cluster_endpoint | jq -r)\"" --inplace constellation-state.yaml
-yq eval ".infrastructure.ipCidrNode =\"$(terraform output ip_cidr_nodes | jq -r)\"" --inplace constellation-state.yaml
-yq eval ".infrastructure.uid =\"$(terraform output uid | jq -r)\"" --inplace constellation-state.yaml
-yq eval ".infrastructure.name =\"$(terraform output name | jq -r)\"" --inplace constellation-state.yaml
-yq eval ".infrastructure.apiServerCertSANs =$(terraform output -json api_server_cert_sans)" --inplace constellation-state.yaml
-```
+<tabItem value="self-managed" label="Self-managed">
 
-</tabItem>
-<tabItem value="azure" label="Azure">
+Self-managed infrastructure allows for more flexibility in the setup, by separating the infrastructure setup from the Constellation cluster management.
+This provides flexibility in DevOps and can meet potential regulatory requirements.
+It's recommended to use Terraform for infrastructure management, but you can use any tool of your choice.
 
 :::info
 
-  If the enforcement policy is set to `MAAFallback` in `constellation-config.yaml`, a manual update to the MAA provider's policy is necessary.
-  You can apply the update with the following commands, where `<VERSION>` is the version of Constellation that should be set up. (e.g. `v2.12.0`)
-
-  ```bash
-  git clone --branch <VERSION> https://github.com/edgelesssys/constellation
-  cd constellation/hack/maa-patch
-  go run . $(terraform output attestationURL | jq -r)
-  ```
+  When using Terraform, you can use the [Terraform module](./terraform-module.md) for ease of use to manage the entire Constellation cluster.
```

Comment:
but the user doesn't know about the CLI, he manages the cluster state through the module. I would like to avoid "create" since it might imply that upgrades are not supported

Reply:
yes, as we discussed